timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/BatchingPaxosLatestSequenceCache.java

```diff
@@ -84,10 +69,15 @@
                 .addAll(cachedEntries.keySet())
                 .build();
         cachedEntries.clear();
-        return unsafeGetLatest(allClients);
+        try {
+            return unsafeGetLatest(allClients);
+        } catch (InvalidAcceptorCacheKeyException e) {
+            log.warn("");
```

Comment:
nit: Throwing an exception isn't a criminal offence - we don't need to read the cache its Miranda rights

I think this is a bug if we get here?

Reply:
it is indeed a bug, will add exception message